packages/iproute/iproute.spec

```diff
@@ -46,7 +46,7 @@ export PKG_CONFIG_PATH='%{_cross_pkgconfigdir}' \\\
 %make_install
 
 install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
-mv %{buildroot}%{_cross_sysconfdir}/iproute2 %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
```

Comment:
https://github.com/iproute2/iproute2/commit/0a0a8f12fa1b03dd0ccbebf5f85209d1c8a0f580 changes installation location to `/usr/lib/iproute2`. It also removes the `CONFDIR` variable.

Above we set `CONFDIR='%{_cross_sysconfdir}/iproute2'` - should we instead make the change `export CONF_USR_DIR='%{_cross_sysconfdir}/iproute2'` in the `%prep%` section?

It also appears we're now expecting conf files in `mv %{buildroot}%{_datarootdir}/iproute2`, that path would evaluate to `/usr/share/iproute2` - I'm not seeing that change in the upstream commit referenced; looks like those conf files will be in `/usr/lib/iproute2` (maybe it came in a different commit?)

Reply:
Yeah I linked the wrong upstream commit on the Commit message. I will fix that. The one to look at is this one -> https://github.com/iproute2/iproute2/commit/962692356a1cc41932e43575f3c50927e455ab53

Upstream is explicitly setting `CONF_USR_DIR?=$(DATADIR)/iproute2` which evaluates to `usr/share/iproute2`. All this is defined in https://github.com/iproute2/iproute2/blob/main/Makefile at the top.